unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -38,18 +45,41 @@ export const MembersContent = ({ account, network, lockAddresses }: Props) => {
       {account && (
         <BrowserOnly>
           <Account network={network} account={account} />
-          <GraphWrapper lockAddresses={lockAddresses} />
+          <GraphWrapper
+            lockAddresses={lockAddresses}
+            signMetadataRequest={signMetadataRequest}
+            accountAddress={account.address}
+          />
         </BrowserOnly>
       )}
     </Layout>
   )
 }
 
-const GraphWrapper = ({ lockAddresses }: { lockAddresses: string[] }) => {
+interface GraphWrapperProps {
+  lockAddresses: string[]
+  signMetadataRequest: typeof signMetadataRequest
+  accountAddress: string
+}
+const GraphWrapper = ({
```

Comment:
Not exactly sure what this should be called but we may want to consider a more meaningful name than GraphWrapper we may not be able to remember what it is supposed to do in 3 weeks

Reply:
Yeah, the general gist is just to avoid bloating one function with too many concerns. I'll think of something better to call this.